src/domain/settlement/index.js

```diff
@@ -91,6 +116,14 @@ module.exports = {
       const settlementWindowsList = await SettlementWindowModel.getBySettlementId({ settlementId }, enums)
       const participantCurrenciesList = await SettlementModel.settlementParticipantCurrency.getParticipantCurrencyBySettlementId({ settlementId }, enums)
       const participants = prepareParticipantsResult(participantCurrenciesList)
+
+      // Build settlement window content array and insert into settlement window list object
+      for (var key of Object.keys(settlementWindowsList)) {
+        settlementWindowsList[key].content = await SettlementWindowContentModel.getBySettlementAndWindowId(settlementId, settlementWindowsList[key].id)
```

Comment:
Instead of deleting keys form the returned list I think it would be clearer to map over the returned list and only add the keys that we want to the response

Reply:
Hi Neal . 
Please review , I dropped the recursive delete object logic and replaced it as you suggested